main.cpp

```diff
@@ -52,84 +52,109 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
+//  Noun: bulldozer
+//  action 1: the blade goes up
+blade.up();
 
-//  action 2:
+//  action 2: the blade goes down
+blade.down();
 
-//  action 3:
+//  action 3:the bulldozer drives forward
+drive.forward();

```

Comment:
the noun for these should be `bulldozer`.  
you've got `blade` as the noun/object in the pseudocde
The code should be written as "Object/Noun . action/function name ();"
- [ ] revise